### DISCUSSION TITLE
Looking for partnership in Brazil

### DISCUSSION THREAD

**Discussion by pedroherpeto** [chatwoot/chatwoot#6722] [Show and tell]
Hi @pranavrajs how are you doing? I'm making some videos with chatwoot on my channel.
I'm looking for an official partnership... Would you be interested in doing this with me?
I think there are many possibilities if I have your official recommendation... here is the official playlist with chatwoot videos
https://www.youtube.com/watch?v=CMOFHvZeWDg&list=PLVvbmF6_9sfzOZOgHm8Md8OAdOvVmPe9T
hope u linke it!
byeeee!

**Comment by pranavrajs** [comment]
@pedroherpeto Thanks for this. I'm not sure about how we can partner (if you have any ideas, let me know). We can definitely get your videos list on our official playlist after a review :)

**Reply by pedroherpeto** [reply]
hello @pranavrajs , everything is fine? I thought we'd do a joint promotion... I produce the content, and you share it on your official channels... and also an authorization that I can put it in my videos, which we have this official support from your team, as a dissemination channel from chatwoot. I don't think about financial compensation, only about producing content and being able to add authority to my content, making it clear that I have your official support.
hugs

**Reply by pedroherpeto** [reply]
Hello @pranavrajs , thank you for sharing my video in your official playlist!
There is a new content with officinal channels of fb, ig e tg: https://youtu.be/XYdZjfQir2o
I will be happy if u could do the same with that! thanksss
